curriculum/challenges/english/blocks/review-python-basics/67f39b40deaec81a3e40e0c5.md

```diff
@@ -18,7 +18,7 @@ dashedName: review-python-basics
 
 ## Variables
 
-- **Declaring Variables**: To declare a variable, you start with the variable name followed by the assignment operator (`=`) and then the data type. This can be a number, string, boolean, etc. Here are some examples:
+- **Declaring Variables**: To declare a variable, you start with the variable name followed by the assignment operator (`=`) and followed by the value. This can be a number, string, boolean, etc. Here are some examples:
```

Comment:
```suggestion
- **Declaring Variables**: To declare a variable, you start with the variable name followed by the assignment operator (`=`) and then the value. This can be a number, string, boolean, etc. Here are some examples:
```